examples/ticks_and_spines/tick-formatters.py

```diff
@@ -0,0 +1,94 @@
+"""
+Show the different tick formatters
+"""
+
+import numpy as np
+import matplotlib.pyplot as plt
+import matplotlib.ticker as ticker
+
+
+# Setup a plot such that only the bottom spine is shown
+def setup(ax):
+    ax.spines['right'].set_color('none')
+    ax.spines['left'].set_color('none')
+    ax.yaxis.set_major_locator(ticker.NullLocator())
+    ax.spines['top'].set_color('none')
+    ax.xaxis.set_ticks_position('bottom')
+    ax.tick_params(which='major', width=1.00, length=5)
+    ax.tick_params(which='minor', width=0.75, length=2.5, labelsize=10)
```

Comment:
Ok, now I get your point. The idea was to emphasize the fact that you can control major / minor ticks / labels separately and the different font sizes really suggest this (at least to me).

Since this is a submission to the gallery, is it important to enforce the defaults ?